packages/connect/src/types/device.ts

```diff
@@ -29,6 +29,15 @@ export enum FirmwareType {
     Regular = 'regular',
 }
 
+export type DeviceState = {
+    // sessionId:
+    // - T1B1 < 1.9.0 passphrase string received from UI
+    // - T2T1 < 2.3.0 result of Deprecated_PassphraseStateRequest message
+    // - otherwise Features.session_id
+    sessionId?: string;
+    external?: string; // 1st address TODO: rename to fingerprint?
```

Comment:
- staticSessionId
- persistentSessionId
- publicId
- staticPublicId

🤔

Reply:
staticSessionId it is